src/Sylius/Behat/Page/Shop/Cart/SummaryPage.php

```diff
@@ -246,59 +265,79 @@ public function waitForRedirect(int $timeout): void
-            'product_total' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-subtotal]',
-            'product_unit_price' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-unit-price]',
-            'product_unit_regular_price' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-regular-unit-price]',
-            'promotion_coupon_validation_message' => '[data-test-cart-promotion-coupon] [data-test-validation-error]',
-            'promotion_total' => '[data-test-cart-promotion-total]',
-            'save_button' => '[data-test-apply-coupon-button]',
+//            'no_taxes' => '[data-test-cart-no-tax]',
+//            'product_row' => '[data-test-cart-product-row="%name%"]',
+//            'product_total' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-subtotal]',
+//            'product_unit_price' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-unit-price]',
+//            'product_unit_regular_price' => '[data-test-cart-product-row="%name%"] [data-test-cart-product-regular-unit-price]',
+//            'promotion_coupon_validation_message' => '[data-test-cart-promotion-coupon] [data-test-validation-error]',
+//            'promotion_total' => '[data-test-cart-promotion-total]',
+            'remove_item' => '[data-test-cart-items] [data-test-cart-item-product-row="%name%"] [data-test-remove-cart-item]',
+//            'save_button' => '[data-test-apply-coupon-button]',
             'shipping_total' => '[data-test-cart-shipping-total]',
-            'tax_excluded' => '[data-test-cart-tax-exluded]',
-            'tax_included' => '[data-test-cart-tax-included]',
-            'update_button' => '[data-test-cart-update-button]',
-            'validation_errors' => '[data-test-validation-error]',
+//            'tax_excluded' => '[data-test-cart-tax-exluded]',
+//            'tax_included' => '[data-test-cart-tax-included]',
+//            'update_button' => '[data-test-cart-update-button]',
+//            'validation_errors' => '[data-test-validation-error]',
         ]);
     }
 
-    /**
-     * @throws ElementNotFoundException
-     */
-    private function hasItemWith(string $attributeName, array|string $selector): bool
-    {
-        $itemsAttributes = $this->getElement('cart_items')->findAll('css', $selector);
+//    /**
+//     * @throws ElementNotFoundException
+//     */
+//    private function hasItemWith(string $attributeName, array|string $selector): bool
+//    {
+//        $itemsAttributes = $this->getElement('cart_items')->findAll('css', $selector);
+//
+//        foreach ($itemsAttributes as $itemAttribute) {
+//            if ($attributeName === $itemAttribute->getText()) {
+//                return true;
+//            }
+//        }
+//
+//        return false;
+//    }
+//
+//    private function getPriceFromString(string $price): int
+//    {
+//        return (int) round((float) str_replace(['€', '£', '$'], '', $price) * 100, 2);
+//    }
```

Comment:
And these methods probably also could be removed

Reply:
The same as above.